index.html

```diff
@@ -25,52 +68,6 @@
     </ol>
     <!-- Wrapper for slides -->
     <div class="carousel-inner" role="listbox">
-       <div class="carousel-navbar">
```

Comment:
This just shows that I removed the navbar from this section (because I moved it up)